koa-ktor-ui/src/main/kotlin/KoaSwaggerUi.kt

```diff
@@ -1,20 +1,14 @@
 package guru.zoroark.koa.ktor.ui
 
-import io.ktor.application.ApplicationCallPipeline
-import io.ktor.application.ApplicationFeature
-import io.ktor.application.application
-import io.ktor.application.call
-import io.ktor.application.feature
+
 import io.ktor.http.ContentType
 import io.ktor.http.HttpStatusCode
 import io.ktor.http.content.OutgoingContent
-import io.ktor.http.content.resourceClasspathResource
 import io.ktor.http.defaultForFileExtension
-import io.ktor.response.respond
-import io.ktor.response.respondText
-import io.ktor.routing.Route
-import io.ktor.routing.application
-import io.ktor.routing.get
+import io.ktor.server.application.*
```

Comment:
Wildcard imports trigger a failure in CI (because this project enforces [ktlint](https://ktlint.github.io) compliance). Would it be possible for you to remove these imports? You should be able to configure IntelliJ to *not* use wildcard imports (Settings -> Editor -> Code Style -> Kotlin -> Imports, then disable all wildcard imports, and empty the "Packages to Use Import with `*`"), then just run an Optimize Imports command (Ctrl + Alt + O)

Reply:
should be fixed in https://github.com/utybo/Koa/pull/7/commits/59815865546e2cdd193077cc41612831ee8c683f